### ISSUE TITLE
Low performance dragging marker on editor

### ISSUE THREAD

**Issue by CristoferCD** [ppy/osu#3598]
Playing with the editor I saw that the bottom timeline can be pretty slow because it´s being updated whenever the event fires.
It could be fixed just by updating the component position on the Update event similar to what is implemented on the TimelinePart.

**Comment by jorolf** [comment]
afaik input events are fired on the same thread as the update event
https://github.com/ppy/osu-framework/blob/master/osu.Framework/Input/InputManager.cs#L231

**Comment by jorolf** [comment]
The low performance probably comes from seeking the clock which causes a lot of transform updates (similar to seeking in a replay)

**Comment by CristoferCD** [comment]
Yes, it's because of that clock. I changed it so it updates a local variable on the drag events and the seek is done on the update event. That makes it way better but I still don't feel it as responsive as the top waveform for example.
